compiler/rustc_codegen_cranelift/src/inline_asm.rs

```diff
@@ -201,114 +200,6 @@ pub(crate) fn codegen_inline_asm_inner<'tcx>(
     call_inline_asm(fx, &asm_name, asm_gen.stack_slot_size, inputs, outputs);
 }
 
-pub(crate) fn codegen_naked_asm<'tcx>(
-    tcx: TyCtxt<'tcx>,
-    cx: &mut crate::CodegenCx,
-    module: &mut dyn Module,
-    instance: Instance<'tcx>,
-    span: Span,
-    symbol_name: &str,
-    template: &[InlineAsmTemplatePiece],
-    operands: &[InlineAsmOperand<'tcx>],
-    options: InlineAsmOptions,
-) {
-    // FIXME add .eh_frame unwind info directives
```

Comment:
is this comment still relevant? We don't do anything with `.eh_frame` in naked functions at the moment, but also unwinding out of asm is UB so I'm not exactly sure what the intention is here?

Reply:
`.eh_frame` is also used for generating backtraces. For naked asm however I realize that only the person writing the assembly can actually write correct unwind info directives.